services/provider/server/server.go

```diff
@@ -224,10 +225,17 @@ func (s *OCSProviderServer) GetStorageConfig(ctx context.Context, req *pb.Storag
 			return nil, err
 		}
 
+		clientMappingConfig, err := s.getClientMapping(ctx)
+		if err != nil {
+			return nil, err
+		}
+		_, hasClientID := clientMappingConfig.Data[req.StorageConsumerUUID]
```

Comment:
I remember the design was a mapping of client names not consumer uuids

Reply:
From provider we are sending the storageConsumerUIDs, and MCO updates this configmap with local ClientID to remote ClientID